packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1411,6 +1413,7 @@ class ListView extends BoxScrollView {
     super.keyboardDismissBehavior,
     super.restorationId,
     super.clipBehavior,
+    bool includeSeparatorForLastItem = false,
```

Comment:
What about if I want a separator before the first item?

I don't think much is gained by adding these parameters; it will just make the API more complex. If one wants this feature, using the regular ListView constructor directly is probably the better way to go.

Reply:
Shall we close this PR and issue then?

Reply:
Sounds good. Thanks!